_posts/0000-01-02-mreifeis.md

```diff
@@ -0,0 +1,9 @@
+---
+
+layout: slide
+
+title: "Welcome Marc to the second slide"
```

Comment:
THis is my second comment and I have no idea what to do next - this totally lacks context!

Reply:
take it easy